src/views/settings/filters/parts/filter-tabs.tsx

```diff
@@ -131,8 +131,9 @@ const FilterTabs: FC = (): ReactElement => {
 					background="gray5"
 					items={tabs}
 					selected={selectedFilterType}
-					onChange={onChange}
```

Comment:
@Zextras/the-dexters this was the change required by the BC of https://github.com/Zextras/carbonio-design-system/pull/126 released with the version 0.5.0 on 14/10. Your change was made on 21/10, so a lot of days after the release of the DS, and it has now introduces a regression.

Reply:
thanks for spotting that out, I will fix it